src/typeahead/typeahead.directive.ts

```diff
@@ -73,8 +74,17 @@ export class TypeaheadDirective implements OnInit, OnDestroy {
   /** should be used only in case typeaheadSingleWords attribute is true.
    * Sets the word delimiter to match exact phrase.
    * Defaults to simple and double quotes.
-   */
+   */  
   @Input() typeaheadPhraseDelimiters = '\'"';
+  /** used to enable/disable the selection on TAB Event.   
+   */
+  @Input() 
+  get selectOnTab(): boolean {
```

Comment:
I believe this option is fine to be only in config, no need for additional input

Reply:
How can a User of the typeahead-directive chose whether the option is selectOnTab = true or false? When putting the directive on an input-element, how can i decide this when there is no input for it?

Reply:
@valorkin @Domainv I still don't know what to do right here. For us it is required that the user can decide whether the selectOnTab is true or false in the HTML for each typeahead instance seperately. Could you please be a little more precise what you expect here? Or tell me an example file in an other component where this is solved the way you imagine it.